test/vanilla/generated/RequiredOptional/operations/explicit.ts

```diff
@@ -31,28 +31,28 @@ export class Explicit {
    * request is sent.
    * @param bodyParameter
    * @param [options] The optional parameters
-   * @returns Promise<Models.ExplicitPostRequiredIntegerParameterResponse>
+   * @returns Promise<coreHttp.RestResponse>
    */
-  postRequiredIntegerParameter(bodyParameter: number, options?: coreHttp.RequestOptionsBase): Promise<Models.ExplicitPostRequiredIntegerParameterResponse>;
+  postRequiredIntegerParameter(bodyParameter: number, options?: coreHttp.RequestOptionsBase): Promise<coreHttp.RestResponse>;
```

Comment:
Changes picked up from regenerating with latest changes in RequiredOptional: https://github.com/Azure/autorest.testserver/commit/e0d8dcad0f06f45ad6ec4416e45b326a139a63ff#diff-a0a97e774df31ce51de76b4a05cea0b7